design/cdk-bootstrap.md

```diff
@@ -0,0 +1,199 @@
+# cdk-bootstrap
+
```

Comment:
Please add a "Non-function requirements" section like https://github.com/aws/aws-cdk/blob/rfc/continuous-delivery/design/cdk-assets.md#non-functional-requirements

Reply:
OK.

Reply:
Any ideas what should be in that section, now that the focus has shifted more towards vending a template?

Reply:
Detailed Description of the template parameters.

Reply:
Removed.